docs/docs/image-filters/runtime-shader.md

```diff
@@ -36,12 +42,88 @@ export const RuntimeShaderDemo = () => {
   return (
     <Canvas style={{ flex: 1 }}>
       <Group>
-         <RuntimeShader source={source} />
-         <Circle cx={r} cy={r} r={r} color="lightblue" />
+        <RuntimeShader source={source} />
+        <Circle cx={r} cy={r} r={r} color="lightblue" />
+      </Group>
+    </Canvas>
+  );
+};
+```
+
+<img alt="Runtime Shader" src={require("/static/img/image-filters/runtime-shader.png").default} width="256" height="256" />
+
+## Supersampling
+
+To keep the output of the image filter crisp, We upscale to filtered drawing to the [pixel density of the app](https://reactnative.dev/docs/pixelratio). Once the drawing is filtered, we scale it back to the original size. This can be seen in the example below. These operations need to be performed on a Skia layer via the `layer` property.
```

Comment:
```suggestion
To keep the output of the image filter crisp, We upscale the filtered drawing to the [pixel density of the app](https://reactnative.dev/docs/pixelratio). Once the drawing is filtered, we scale it back to the original size. This can be seen in the example below. These operations need to be performed on a Skia layer via the `layer` property.